py-bns.py

```diff
@@ -58,6 +71,13 @@ def connect(self):
         # save access token for later use
         self.access_token = str(response.json()['access_token'])
 
+    def get_story(self):
```

Comment:
the `story_id` should be an argument passed to the function. `get_story(self, story_id)`. So that when I invoke it later I can tell it which story I want to grab: `bb.get_story('P6ZNX36JIJUP')`